src/development/sic-calls/history.md

```diff
@@ -25,6 +26,38 @@
+[Video recording](TBD)
+
+### Team updates
+- [@gballet](https://x.com/gballet) ([@go_ethereum](https://x.com/go_ethereum)) is assisting [@CPerezz19](https://x.com/CPerezz19) in reactivating execution spec tests to be binary-tree compatible and proposed a new gas-accounting PR (EIP-8032) aimed at stemming state growth.
+- [@CPerezz19](https://x.com/CPerezz19) is working with [@gballet](https://x.com/gballet) on binary-tree test fixes (state diffs and transitions), has identified and fixed bugs in the binary-tree implementation, and continues BloatNet work while pushing benchmarks to EELS for integration with Camille’s tool from [@NethermindEth](https://x.com/NethermindEth).
+- Thomas Zamojski ([@HyperledgerBesu](https://x.com/HyperledgerBesu)) shared that [@kt2am1990](https://x.com/kt2am1990) is working on the transition and awaiting binary-tree test returns. Thomas has been focused on gas optimization and will return to statelessness, picking up the transition topic from Karim.
+- [Gabriel Rocheleau](https://x.com/gabrocheleau) ([EthereumJS](https://twitter.com/EFJavaScript)) shared that EthereumJS could consider supporting stateless clients for future testnet usages, but absence of execution witnesses & proofs on binary tree testnets makes this impossible for the time being.
+
+### Updates on binary trie t8n self-fixtures consumption progress. ([@CPerezz19](https://x.com/CPerezz19))
+- State diffs are the primary blocker; more than half of the 500+ tests pass when excluding state diffs.
+- Temporary removal of state-diff tests to simplify rebases and speed PR merges.
+- Fixture refactor to align vertical and binary trees; transition to EELS with a Python reimplementation to generate fixtures natively.
+- Test representation moving from separate allocs to a unified key–value list; adopt SHA256 for simpler key computation.
+
+### Discussion on code chunks inclusion in the tree ([@CPerezz19](https://x.com/CPerezz19))
+- Proposal to store code chunks separately and deduplicate across accounts, removing the preferential treatments of the first 128 chunks in the account leaf. 
+- Treat code and storage uniformly to simplify the binary-tree structure; decision postponed pending further research.
+
+### BloatNet updates ([@CPerezz19](https://x.com/CPerezz19))
+- Unblocked on merging PRs to EELS; submitted fixes on stop-contract handling and benchmarking.
+- Multi-opcode tests in place; key single-opcode tests focus on L1 scaling (SSTORE on the largest ERC-20s; SLOAD into empty/likely-empty slots).
+- Goal is to integrate with Camille’s benchmarking tool from [@NethermindEth](https://x.com/NethermindEth); cross-team call scheduled. Stop contracts not specifiable across tests remains a major issue.
+- Syncing tests: collaboration with partners; clients like Reth and Erigon are actively debugging syncing/PD issues. Several benchmark runs executed; target is five clients in one pass to avoid heavy snapshot re-runs.
```

Comment:
also not sure what PD/issues are?

Reply:
removed PD, it was in the transcript but can be left out, issues is informative enough